### DISCUSSION TITLE
Email Verification in Enteprise Server

### DISCUSSION THREAD

**Discussion by y4n9squared** [community/community#30926] [Enterprise] [answered]
After upgrading to Github Enterprise Server 3.6.0, no member of the organization can participate in Discussions. Following the "verify your email address" link takes you to your User Settings page, where there doesn't seem to be any way to do this.

![Screen Shot 2022-08-26 at 9 49 38 AM](https://user-images.githubusercontent.com/3999513/186922003-df056a9a-e90a-458f-a879-32d86931b437.png)

**Comment by stewvatic** [comment, marked as answer]
Confirming this is resolved in GHE v3.6.1
